docker-compose-rabbitmq.yml

```diff
@@ -0,0 +1,17 @@
+services:
```

Comment:
move it to `docker-compose-db.yaml` and `docker-compose.yaml`, we don't need a separate file just for `RabbitMQ`